2.15/2.15.8/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.15.8
+Скопируй весь код из задачи 2.10.8 и переделай его - теперь должна быть
+отдельная функция getPoints, с помощью которой мы будем вычислять результат.
+В итоге код должен получиться следующим:
+
+function getPoints() { ... } // часть функционала находится здесь
+
+let result = Number(prompt('Введите стартовые очки'));
+
+for (let i = 0; i < 3; i++) {
+result += getPoints();
+}
+
+alert(У вас ${result} очков);
+
+### 2.10.8
+Появляется вопрос: "Введите стартовые очки"
+Пользователь вводит какое то число. Потом появляется новый вопрос:
+"Добавить или отнять?"
+Если пользователь нажал "ОК", то к стартовым очкам добавляется 10.
+Если нажал "Отмена", то от стартовых очков отнимается 10.
+И так 3 круга, а потом выводится сообщение:
+"У вас Х очков"
+<script>
+	function getPoints() {
+		const shouldIncreasePoints = confirm('Добавить или отнять?');
+
+		const points = (shouldIncreasePoints) ? +10 : -10;
```

Comment:
you can remove the symbol "+" before 10, we don't add this symbol for positive numbers